app/js/components/setsOrderEntry/OrderSetForm.jsx

```diff
@@ -0,0 +1,87 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import SelectBox from '../selectBox';
+import Accordion from '../orderEntry/Accordion';
+import OrderHeader from '../orderEntry/OrderHeader';
+import OrderSetDetails from './OrderSetDetails';
+import './styles.scss';
+
+
+class OrderSetForm extends PureComponent {
+  state = {
+    items: [{ value: 'Choose a set to add', id: 0 }, { value: 'Mineral', id: 1 }],
+    selectedItem: '',
+    isDisabled: false,
+    displaySets: 'block',
+    displayForm: 'none',
+  }
+
+  onSelectOrderSet = () => {
+    this.setState({ displaySets: 'none', displayForm: 'block' });
+  }
+
+  render() {
+    const {
+      items, selectedItem, isDisabled, displaySets, displayForm,
+    } = this.state;
+    return (
+      <React.Fragment>
+        <div className="order-set-entry">
+          <div>
+            <h1>Order from Sets</h1>
+          </div>
+          <span style={{ display: displaySets }}>
+            <SelectBox
+              items={items}
+              selectedItem={selectedItem}
+              onChange={this.onSelectOrderSet}
+            />
+          </span>
+          <span style={{ display: displayForm }}>
+            <div className="set-header">
+              <h1>DIABETES INITIAL WORKUP</h1>
+            </div>
+            <table>
+              <tbody>
+                <tr className="set-details">
+                  <th className="set-details-header">Details</th>
+                  <th className="set-actions-header">Actions</th>
+                </tr>
+                <Accordion
+                  title={
+                    <OrderHeader
+                      handleEdit={() => {}}
+                      handleDiscontinue={() => {}}
```

Comment:
Why are these functions that return empty bodies? Are they beyond scope?

Reply:
They are plaecholders for now will be populated when the functionality is ready

Reply:
*placeholders